docs/how-tos/python_consume.md

```diff
@@ -0,0 +1,139 @@
+# Consume GraphQL API using Python
```

Comment:
Think we could be a bit snappier with this title :sweat_smile: Perhaps `Python Consumer`?